package.json

```diff
@@ -4,11 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "node  --harmony server.js",
```

Comment:
Node v7.6.0 was released today with `async/await` support by default. If we upgrade to that, we should be able to remove the `--harmony` flags here and in the test commands below.